exercises/mark-griffith/short-response.md

```diff
@@ -0,0 +1,23 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+    
+     A Parameter is variable in the declaration of function, it is where an arguement would be stored.
+
+     An Argument is the actual value of this variable that gets passed to function.
+
+2. What happens when you invoke a JavaScript function with a missing argument? What about calling a function with _extra_ arguments?
+3. 
+     Invoking a JavaScript function with missing arguement would run but return either undefined since thats the value that is placed in its stance.
```

Comment:
Not correct here. The function won't _return_ undefined. It will simply pass `undefined` as the value for any missing argument. Make this change.